CHANGELOG.md

```diff
@@ -38,6 +38,10 @@ title: Changelog
 * Upgrade from `@custom-elements-manifest/analyzer@0.4.1` to `@custom-elements-manifest/analyzer@0.6.4`.
 * Fix Custom Element Manifest generation web dev server plugin: disable caching
 * Add new eslint plugin `eslint-plugin-lit-a11y` to prevent accessibility issues.
+* Add new dependency `@open-wc/testing` to allow Web Test Runner to run tests for accessibility issues.
+* Add new dependency `@web/test-runner-commands` so that Web Test Runner can test both on desktop and mobile.
+* Add new helpers to extract stories and run accessibility test on them.
+* Add new test files in most of the component folders. These test files only contain accessibility tests for the moment.
```

Comment:
This commit contains both manual updates and automated updates (with the script). Next time, I would prefer these in two commits so the automated modifications are isolated. It's also easier if you need to update them during the review.